jiant/utils/tokenization_normalization.py

```diff
@@ -66,6 +68,10 @@ def normalize_tokenizations(
         space_tokenization = [token.lower() for token in space_tokenization]
         modifed_space_tokenization = bow_tag_tokens(space_tokenization)
         modifed_target_tokenization = _process_sentencepiece_tokens(target_tokenization)
+    # no-op for test cases that use a dummy tokenizer
+    elif isinstance(tokenizer, TestTokenizer):
+        modifed_space_tokenization = space_tokenization
+        modifed_target_tokenization = target_tokenization
```

Comment:
Haven't taken too close a look - is there a way to get around importing test code?

Reply:
Yeah — 60d11ad is an approach where we temporarily mock an instance of a tokenizer supported by this code.